test/IsolatedKernelTestCase.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Matthias\BundlePlugins\Tests;
+
+/**
+ * Class IsolatedKernelTestCase
+ *
+ * @package Matthias\BundlePlugins\Tests
+ */
+class IsolatedKernelTestCase extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var string Directory with symfony cache
+     */
+    private $tempDirectory;
+
+    private $usedIdentifiers = array();
+
+    /**
+     * Sets a temporary directory for storing symfony cache
+     */
+    public function setUp()
+    {
+        parent::setUp();
+        $this->tempDirectory = tempnam(sys_get_temp_dir(), 'symfony-phpunit');
+
+        // We don't want a temporary _file_, just a nice and unique path
+        // If you can't stand this uglyness, please provide a pull request ;-)
+        unlink($this->tempDirectory);
```

Comment:
Well, I usually just use `uniqid()` to get a unique string at least, then just do a `mkdir()` (with the recursive option).